sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -335,7 +335,10 @@ def set_named_target(self, name):
             for n, this_name in enumerate(joints):
                 if this_name in active_names:
                     js[this_name] = positions[n]
-            self._move_group_commander.set_joint_value_target(js)
+            try:
+                self._move_group_commander.set_joint_value_target(js)
+            except Exception as e:
+                print(e)
```

Comment:
Why not `rospy.logerr`?

Reply:
sorry yes changing now was debugging too quick haha. will do rosinfo as it isn't really an error?

Reply:
ok